custom/ilsgateway/tanzania/handlers/soh.py

```diff
@@ -39,19 +38,26 @@ def parse_report(val):
     [('zi', 10), ('co', 20), ('la', 30)]
     """
 
-    def _cleanup(s):
-        return six.text_type(s).encode('utf-8')
+    if six.PY3:
+        maketrans = str.maketrans
+    else:
+        from strop import maketrans
+
+    if six.PY3:
+        val = six.text_type(val)
+    else:
+        val = six.text_type(val).encode('utf-8')
 
     return [
-        (x[0], int(x[1].translate(maketrans("lLO", "110"))))
+        (x[0], int(x[1].replace(' ', '').translate(maketrans("lLO", "110"))))
```

Comment:
This appears to be new functionality, not a py2 -> py3 compatibility change. Can you explain the reason for it or what I'm missing?

Reply:
This was a breaking change in python 2, so I instead decided to fork the function.  The py3 tests pass, so the python 3 functionality should be preserved.